components/ContactDetailsHeader/ContactDetailsHeader.styled.js

```diff
@@ -29,3 +29,18 @@ export const BirthdayContainer = styled.div`
   gap: 5px;
   margin-top: 10px;
 `;
+
+export const ImageContainer = styled.div`
+  position: relative;
+`;
+
+export const UploadImageIcon = styled(Image)`
+  position: absolute;
+  top: -9px;
+  right: -6px;
+  cursor: pointer;
+  background-color: white;
+  border: 1px solid lightgrey;
```

Comment:
It's suggested to use color hexcodes instead of the color names since it provides. more control and flexibility.

Reply:
Thanks. #D3D3D3